hexrd/instrument.py

```diff
@@ -112,6 +115,9 @@
 
 distortion_registry = distortion_pkg.Registry()
 
+multi_ims_key = 'mulitple'
```

Comment:
Maybe we can make this key appear in one place, such as constants, so that we don't have to repeat it twice? It is both here and in the `hexrd.config.imageseries` file.